modules/mongodb/config.go

```diff
@@ -0,0 +1,15 @@
+package mongo
```

Comment:
My suggestion is to move Config to the `mongodb.go`. Why do you think having it in a separate file is better? (after removing python-config-way code)

Reply:
I've moved the code to `mongodb`